src/eq_schema/schema/Submission/index.js

```diff
@@ -0,0 +1,13 @@
+class Submission {
+  constructor(postSubmission) {
+    this.button = "Submit";
+    if (postSubmission.viewPrintAnswers) {
+      this.guidance =
+        "You will have the opportunity to view, save or print a copy of your answers after submitting this survey.";
+    }
+    this.title = "Submit your questionnaire";
+    this.warning = "You cannot view your answers after submission";
+  }
+}
+
+module.exports = Submission;
```

Comment:
```suggestion
class Submission {
  constructor(submission) {
    this.button = "Submit";
    if (submission.viewPrintAnswers) {
      this.guidance =
        "You will have the opportunity to view, save or print a copy of your answers after submitting this survey.";
    }
    this.title = "Submit your questionnaire";
    this.warning = "You cannot view your answers after submission";
  }
}

module.exports = Submission;
```

Renamed parameter

Reply:
Post submission and submission are two different pages. 

Submission represents the page in Runner that is shown before hitting the submit button - this page would include the hub if it is enabled.

Post submission represents the page in Runner that is shown after hitting the submit button - this page is setup in Publisher and is based on data populated in Author.

The page edited in this PR is the Submission page, instead of the Post Submission page.